.github/workflows/docker-bases.yml

```diff
@@ -31,6 +31,18 @@ jobs:
             dockerfile: './docker/Dockerfile.nvidia'
             runner: ["self-hosted", "gpu", "docker", "openmp"]
 
+          - tag: 'devitocodes/bases:nvidia-nvcc'
```

Comment:
will these be tested somehow?

Reply:
only in devitopro because there's no CUDA in devito (yet)